.github/workflows/wheels.yml

```diff
@@ -15,6 +15,13 @@ jobs:
 
     steps:
 
+      - name: Prune stale docker containers
+        run: |
+          # If cibuildwheel crashes (or, say, is OOM-killed), it leaves behind a
+          # docker container.  Eventually these consume all the disk space on
+          # this machine.
+          docker container prune -f
```

Comment:
could we put this at the end of `llvm-build.yml` since this is where the images are created?

Reply:
Ah, actually you are right. Both workflows can leave behind stale docker containers. Let me prune containers as well in the `llvm-build` workflows